rfc/2022/2022-07-13 - Platform Wide Documentation vs Component Specific Documentation Locations.md

```diff
@@ -0,0 +1,88 @@
+
+## Background
+
+The Platform currently has two centralized locations for documentation:
+* The (Platform website)[https://depo-platform-documentation.scrollhelp.site/index.html], which uses (Scroll Viewport)[https://www.k15t.com/software/scroll-viewport/scroll-viewport-for-confluence-cloud] to display specific content hosted in Confluence. This website hosts the following types of content:
+    * **“Getting started”** content for onboarding VFS team members to work on the Platform
+    * **“Developer docs”** content about writing and submitting code for review
+    * **“Research and design”** content for adhering to the Platform’s guidance for research and design
+    * **“Analytics and insights”** content, such as for analytics, KPI dashboards, and monitoring
+    * **“Collaboration cycle”** content
+    * **“Support”** policies and incident management
+    * **“User guides, how-to instructions, tutorials, or reference documents for VFS teams”** ((source)[https://vfs.atlassian.net/wiki/spaces/CUI/pages/2186543112/RFC+DRAFT+Documentation+Locations+-+Console+UI])
+* (Operational documentation)[https://vfs.atlassian.net/wiki/spaces/AP/pages/1287782439] throughout Confluence, which includes internal content such as “(meeting notes, roles and responsibilities documentation, decision documentation, [and] initiative outlines.)[https://vfs.atlassian.net/wiki/spaces/AP/pages/1287782439]”
+
+While Confluence and the Platform website contain helpful documentation for VFS (and Platform) teams, a lot of documentation—specifically technical documentation for a software component, service, or infrastructure resource—is found throughout various GitHub repositories. This documentation is vital to developers trying to troubleshoot or set up software dependencies, but it is much harder to find. Developers typically find it by asking around, receiving a few different answers, and sometimes finding different versions of the documentation. When they find the documentation, it is often difficult to know whether it is complete or up to date. Additionally, some VFS teams rely on systems external to the VA and have had difficulty finding or maintaining documentation about these systems.
+
+## Motivation
+
+If we can make it easy for developers to locate and store technical documentation, we can save them hours (or days) of extra work. In addition, having a consistent method and location for finding this documentation is a crucial part of improving the experience and efficiency of VFS teams.
+
+## Design
+
+We propose that the central location for viewing component-specific technical documentation—including APIs, software components, and infrastructure—should be the Console catalog. The catalog makes it easy for users to navigate an org chart-type view of the VA ecosystem (including VFS apps, Platform services, and user groups) and quickly find documentation about any part of it.
```

Comment:
How does design system documentation fit into this? We need to keep it public and independent of the Console, but it seems that it would fit within your definition of "component-specific technical documentation" here.

Reply:
Since it is not specific to a single software entity—and is a resource for all VFS teams—there's no reason for Design System documentation to move at all.

Reply:
Should that be indicated here?

Reply:
> Should that be indicated here?

Will do.